### PR TITLE
Remove tests that facilitated refactor + tag patch release

### PR DESCRIPTION
**Pull request by hannahilea**
Follow-up to #70: make a patch release (no breaking API changes) and remove tests that facilitated refactor.

Additionally, moved slew of metrics functions from `src/learn.jl` to `src/metrics.jl`; moved only, no code changes.

### CODE REVIEW

File: src/learn.jl [hunk 1]
```diff
@@ -240,530 +240,6 @@ function evaluate!(predicted_hard_labels::AbstractVector,
...
-    has_value(optimal_threshold_class) || return nothing
-    length(classes) == 2 ||
-        throw(ArgumentError("Only valid for binary classification problems"))
-    optimal_threshold_class in Set([1, 2]) ||
-        throw(ArgumentError("Invalid threshold class"))
-    return nothing
-end
-
-"""
-    evaluation_metrics_row(observation_table, classes, thresholds=0.0:0.01:1.0;
```
Comment (hannahilea) [inline src/learn.jl:512]: This is the only function that was NOT moved; it was deleted outright, and replaced by the (now-renamed) `refactored_evaluation_metrics_row` in src/metrics.jl.

File: src/metrics.jl [hunk 2]
```diff
@@ -548,3 +610,298 @@ function _evaluation_row(tradeoff_metrics_table, hardened_metrics_table,
                          optimal_threshold_class, class_labels, thresholds,
                          optimal_threshold, stratified_kappas)
 end
+
+#####
+##### Pipeline helper functions
```
Comment (hannahilea) [inline src/metrics.jl:615]: all moved verbatim from src/metrics.jl

File: src/plotting.jl [hunk 3]
```diff
@@ -493,3 +519,55 @@ plot_roc_curves(args...; kw...) = axisplot(plot_roc_curves!, args; kw...)
                 annotation_text_size=20)
 """
 plot_kappas(args...; kw...) = axisplot(plot_kappas!, args; kw...)
+
+#####
+##### Deprecation support
+#####
+
+"""
+    evaluation_metrics_plot(predicted_hard_labels::AbstractVector,
```
Comment (hannahilea) [inline src/plotting.jl:528]: moved verbatim from src/learn.jl

**Comment by hannahilea** [comment]
@ericphanson if you think the code moving is a bad idea, I can move it back!!

**Review by ericphanson — APPROVED** [review]
nice! I think moving is a good idea

**Pull request merged**